variables.tf

```diff
@@ -84,6 +84,11 @@ variable "github_admins" {
   description = "A list of Github teams that should have admins access"
 }
 
+variable "github_organization" {
+  type        = string
+  description = "The Github organization to connect the workspace to"
+}
```

Comment:
As attaching the workspace to a VCS is optional, `github_organization` needs a `null` default so that it's not a required variable.

Reply:
Updated, tnx!